README.md

```diff
@@ -24,12 +24,24 @@ Por el contrario, una máquina virtual (VM) ejecuta un sistema operativo "invita
 
 ## Mi primer contenedor
 ```bash
+# Para descargar una image sin ejecutarla
+docker pull <image_name>
+
 # Para lanzar mi primer el contenedor
+docker run --name my-first-container hello-world
+
+# Para lanzar mi primer el contenedor desacoplado de la terminal.
 docker run -d --name my-first-container hello-world
 
+# Para lanzar mi primer el contenedor desacoplado de la terminal .
+docker run -d --name my-first-container -p <desired_port>:<Source_port> hello-world
+
 # Para listar mis contenedores
 docker ps
 
+# Para listar mis contenedores aun asi no esten ejecutandose
+docker ps -a 
```

Comment:
This commando could be better in another doc like frequent commands.. I think the docker image ls command too.